src/TodoListItem.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react'
-
-function TodoListItem(props) {
-    console.log(props)
+function TodoListItem({todo}) {
+    console.log({todo})
```

Comment:
todo doesn't need to be in curly brackets since you're printing out the variable as a whole and not printing out it's value like in line 6 with {todo.title}